src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/StreamHelper.cs

```diff
@@ -49,43 +48,19 @@ internal WebResponseContentMemoryStream(Stream stream, int initialCapacity, Cmdl
 
         /// <summary>
         /// </summary>
-        public override bool CanRead
-        {
-            get
-            {
-                return true;
-            }
-        }
+        public override bool CanRead => true;
 
         /// <summary>
         /// </summary>
-        public override bool CanSeek
-        {
-            get
-            {
-                return true;
-            }
-        }
+        public override bool CanSeek => true;
 
         /// <summary>
         /// </summary>
-        public override bool CanTimeout
-        {
-            get
-            {
-                return base.CanTimeout;
-            }
-        }
+        public override bool CanTimeout => base.CanTimeout;
```

Comment:
I wonder why we need this at all. This is the same as overrides of parameters in web cmdlets - we can remove them there too.

Reply:
done